contracts/ColonyNetworkStorage.sol

```diff
@@ -26,59 +26,62 @@ import "./IMetaColony.sol";
 
 contract ColonyNetworkStorage is CommonStorage, ColonyNetworkDataTypes, DSMath {
   // Number of colonies in the network
-  uint256 colonyCount;
+  uint256 colonyCount; // Storage slot 6
   // uint256 version number of the latest deployed Colony contract, used in creating new colonies
-  uint256 currentColonyVersion;
+  uint256 currentColonyVersion; // Storage slot 7
   // Address of the Meta Colony
-  address metaColony;
+  address metaColony; // Storage slot 8
   // Address of token locking contract
-  address tokenLocking;
+  address tokenLocking; // Storage slot 9
   // Network fee inverse value, e.g 5% => 100/5=20, 1% => 100/1=100 etc.
-  uint256 feeInverse;
-  // Maps index to colony address
-  mapping (uint256 => address) colonies;
-  mapping (address => bool) _isColony;
-  // Maps colony contract versions to respective resolvers
-  mapping (uint256 => address) colonyVersionResolver;
-  // Contains the address of the resolver for ReputationMiningCycle
-  address miningCycleResolver;
-  // Contains all global and local skills in the network, mapping skillId to Skill. Where skillId is 1-based unique identifier
-  mapping (uint256 => Skill) skills;
+  uint256 feeInverse; // Storage slot 10
   // Number of skills in the network, including both global and local skills
-  uint256 skillCount;
+  uint256 skillCount; // Storage slot 11
   // skillId of the root global skills tree
-  uint256 rootGlobalSkillId;
+  uint256 rootGlobalSkillId; // Storage slot 12
 
-  // Address of the currently active reputation mining cycle contract
-  address activeReputationMiningCycle;
-  // Address of the next active reputation mining cycle contract, which is where new reputation updates are put.
-  address inactiveReputationMiningCycle;
   // The reputation root hash of the reputation state tree accepted at the end of the last completed update cycle
-  bytes32 reputationRootHash;
+  bytes32 reputationRootHash; // Storage slot 13
   // The number of nodes in the reputation state tree that was accepted at the end of the last mining cycle
-  uint256 reputationRootHashNNodes;
+  uint256 reputationRootHashNNodes; // Storage slot 14
+
+  // Contains the address of the resolver for ReputationMiningCycle
+  address miningCycleResolver; // Storage slot 15
+  // Address of the currently active reputation mining cycle contract
+  address activeReputationMiningCycle; // Storage slot 16
+  // Address of the next active reputation mining cycle contract, which is where new reputation updates are put.
+  address inactiveReputationMiningCycle; // Storage slot 17
+
+  // Maps index to colony address
+  mapping (uint256 => address) colonies; // Storage slot 18
```

Comment:
Interesting, I vaguely recall learning that for mappings and arrays, the storage slot is elsewhere? Some function of the hash of the mapping and key names?

Reply:
The hash is based on the key and the storage slot that is 'used' (see [the docs](https://solidity.readthedocs.io/en/v0.4.24/miscellaneous.html?highlight=slot#layout-of-state-variables-in-storage)).